
Header header

# same as SetJoystickMode.srv request
FlightCommandMode mode

# same as JoystickAction.srv request
JoystickParams cmd
